modules/subxt/utils/src/lib.rs

```diff
@@ -149,16 +149,16 @@ mod gargantua_conversion {
 		}
 	}
 
-	impl From<ismp::router::Post> for runtime_types::ismp::router::Post {
-		fn from(post: ismp::router::Post) -> Self {
+	impl From<ismp::router::PostRequest> for runtime_types::ismp::router::Post {
+		fn from(post: ismp::router::PostRequest) -> Self {
 			Self {
 				source: post.source.into(),
 				dest: post.dest.into(),
 				nonce: post.nonce,
 				from: post.from,
 				to: post.to,
 				timeout_timestamp: post.timeout_timestamp,
-				data: post.data,
+				data: post.body,
```

Comment:
reminder to regenerate runtime types before we merge, so we can fix this conversion.